tests/pb/test_decoder.cc

```diff
@@ -558,9 +571,9 @@ uint32_t Hash(const string& proto, const string* expected_output, size_t seam1,
 
 void CheckBytesParsed(const upb::pb::Decoder& decoder, size_t ofs) {
   // We could have parsed as many as 10 bytes fewer than what the decoder
-  // previously accepted, since we can buffer up to 10 partial bytes internally
+  // previously accepted, since we can buffer up to 12 partial bytes internally
```

Comment:
Lots of different numbers 10, 12, 14 
should they all be 14?  Maybe with explanation on where 14 was calculated